node_modules/dreamhost-css/package.json

```diff
@@ -1,26 +1,27 @@
 {
-  "_from": "dreamhost-css@^0.2.5",
+  "_from": "dreamhost-css@latest",
```

Comment:
Did you update this manually? I wonder if it will be overridden the next time someone reversions.

Reply:
No it happened automatically when i updated